ci/generate_docs.sc

```diff
@@ -197,9 +213,28 @@ def launchPreview(buildDir: Path): Unit = {
   bindingFuture.flatMap(_.unbind()).onComplete(_ => system.terminate()) // and shutdown when done
 }
 
-def buildDocs(buildDir: Path, docsDir: Path) = {
+def copyMarathon(buildDir: Path): Path = {
+  println(s"Copying marathon folder into $buildDir")
+  cp.into(pwd/up, buildDir)
```

Comment:
Why use this instead of a temp folder?

Reply:
well, build dir is a temp folder already :)